### PR TITLE
detail video

### PR DESCRIPTION
**Pull request by DongHien0896**
UI & get data screen detail cast: 
https://edu-redmine.framgia.vn/issues/1302

### CODE REVIEW

**Comment by DongHien0896** [comment]
ready

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -26,6 +26,8 @@
         <activity
             android:name="com.framgia.hien.moviedb.screen.main.MainActivity"
             android:label="@string/app_name" />
-        <activity android:name="com.framgia.hien.moviedb.screen.play.PlayActivity"/>
+        <activity android:name="com.framgia.hien.moviedb.screen.play.PlayActivity" />
+        <activity android:name="com.framgia.hien.moviedb.screen.person.PersonActivity"/>
     </application>
-</manifest>
+
+</manifest>
```
Comment (tuannguyen38) [inline app/src/main/AndroidManifest.xml]: need an endline

File: app/src/main/java/com/framgia/hien/moviedb/screen/detail/DetailViewModel.java [hunk 2]
```diff
@@ -203,7 +204,13 @@ public void onItemClick(int companyId) {
 
     @Override
     public void onItemCastClick(int castId) {
-        Toast.makeText(mActivity.getApplicationContext(), Constants.TYPE_NOW_PLAYING, Toast.LENGTH_LONG).show();
+        if (castId == DEFAULT_VALUE) {
+            Toast.makeText(mActivity.getApplicationContext(), Constants.TYPE_NOW_PLAYING, Toast.LENGTH_LONG).show();
+            return;
+        }
+        Intent intent = new Intent(mActivity, PersonActivity.class);
```
Comment (tuannguyen38) [inline app/src/main/java/com/framgia/hien/moviedb/screen/detail/DetailViewModel.java]: convention 2.11

**Comment by DongHien0896** [comment]
ready

**Pull request merged**